src/Components/FollowButton/FollowButton.tsx

```diff
@@ -40,35 +38,36 @@ export class FollowButtonDeprecated extends React.Component<Props, State> {
         onMouseEnter={() => this.setState({ showUnfollow: true })}
         onMouseLeave={() => this.setState({ showUnfollow: false })}
       >
-        {text}
+        <Sans size="2" weight="medium">
```

Comment:
Consider [the updated type treatments](https://palette.artsy.net/tokens/typography/)? — `<Text variant="mediumText">`

Reply:
Ya this should change - switching this to use the `Button` instead, which is what we're doing in Force
<img width="445" alt="Screen Shot 2020-09-30 at 2 48 20 PM" src="https://user-images.githubusercontent.com/1497424/94727248-6ef61f00-032c-11eb-8f8c-93d067131cce.png">

In general though, lots of old text, text styles that are pre-palette in Reaction -- articles could definitely use a typography update.